docs/02-install.md

```diff
@@ -144,6 +151,92 @@ file in your installation artifacts directory and simply re-export it:
 
     export KUBECONFIG=/path/to/something/auth/kubeconfig
 
+# Advanced Installations
+While the OpenShift 4 installer's purpose in life is to streamline operations
+in order to guarantee success, there are a few options that you can adjust by
+using a configuration file. Namely, you can change the default number of
+instances of workers and masters, and you can change both the master and
+initial worker EC2 instance types.
+
+## Generate Installer Configuration File
+This example will use the `--dir` option. The first step is to ask the
+installer to pre-generate an installer configuration file,
+`install-config.yaml`:
+
+```sh
+openshift-install --dir /path/to/something create install-config
+```
+
+After following the same interactive prompts you saw earlier when performing
```

Comment:
After following the same interactive prompts as [basic-installation],

Reply:
I don't think we need to link on the same page? That seems excessive?